dwave/plugins/torch/samplers/_base.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2025 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+import copy
+
+import torch
+
+from dwave.plugins.torch.models.boltzmann_machine import GraphRestrictedBoltzmannMachine
+
+
+__all__ = ["TorchSampler"]
+
+
+class TorchSampler(abc.ABC):
```

Comment:
Not a fan of the name `TorchSampler` because `Torch` is redundant. If the motivation for naming it `TorchSampler` is to distinguish it from `dimod.Sampler`, that can be addressed with, for example, `import Sampler as TorchSampler`.